fb-install-foreman.bats

```diff
@@ -96,6 +96,7 @@ setup() {
 @test "configure repository" {
   if tIsRedHatCompatible; then
     if [ -n "$FOREMAN_CUSTOM_URL" ]; then
+      # use a temporary repo definition as the foreman-release EVR isn't known
       cat > /etc/yum.repos.d/foreman-custom.repo <<EOF
```

Comment:
Ok, but then you can use enabled=0 in the definition instead of using yum-config-manager.